source/mainnet/smart-contracts/guides/build-schema.rst

```diff
@@ -61,14 +99,20 @@ optional ``parameter`` and ``error`` attributes for the
    #[derive(Serial, Reject, SchemaType)]
    enum InitError { ... }
 
-   #[init(contract = "my_contract", parameter = "InitParameter", error = "InitError")]
+   #[init(contract = "my_contract", parameter = "InitParameter", error = "InitError", event = "Event")]
```

Comment:
I can't see the event part without scrolling, so please introduce some line breaks.

And please add an `enum Event`, similar to `InitError`, so there are no unknown data types in here.